docker-compose-fs.yml

```diff
@@ -132,24 +93,12 @@ services:
     mem_limit: 1024m
 
 volumes:
-  data_pgsql:
-    external:
-      name: ${BATS_PGSQL_VOLUME_NAME}
   data_ems:
     external:
       name: ${BATS_EMS_STORAGE_VOLUME_NAME}
-  data_es_1:
-    external:
-      name: ${BATS_ES_1_VOLUME_NAME}
-  data_es_2:
```

Comment:
An idea about the memory used by loading the db and the es data in memory?

Reply:
Removing data volumes has no effect on container memory usage.  
Files has been store in the container instead of the volume, so we lost data when we restart the container.